universe/com.goal_smart/manifest.tt

```diff
@@ -14,42 +45,107 @@ class @com.goal_smart
+  #_[canonical="league losers, league last, league bottom"]
+  #_[confirmation="last place of $league"]
+  #_[result = ["${team} is in last place with ${points} points"]]
+  #[poll_interval=3600000ms]
+  #[doc="Get the last place team of a league"];
+
+  monitorable query teamRank(in req team_id: Entity(com.goal_smart:soccer_team) #_[prompt="what team do you want out check out"],
+                        out team : Entity(com.goal_smart:soccer_team),
+                        out rank: Number,
+                        out league: Entity(com.goal_smart:soccer_league),
+                        out points: Number)
+  #_[canonical="team standings, team rank, team rankings, team performance"]
+  #_[confirmation="league standings of $team"]
+  #_[result = ["${team} is currently number ${rank} in the ${league} with ${points} points"]]
+  #[poll_interval=3600000ms]
+  #[doc="Get the league standings for a team"];
+
+  monitorable list query relegationTeams(in req league_id: Entity(com.goal_smart:soccer_league) #_[prompt="what league standings do you want out check out"],
+                        out team : Entity(com.goal_smart:soccer_team),
+                        out rank: Number,
+                        out points: Number)
+  #_[canonical="league relegation, league bottoms, relegation teams, relegation sides"]
+  #_[confirmation="relegation teams of $league"]
+  #_[result = ["${rank} . ${team} . Points: ${points}"]]
+  #[poll_interval=3600000ms]
+  #[doc="Get the relegation teams of a league"];
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+  monitorable list query teamFixtures(in req team_id: Entity(com.goal_smart:soccer_team) #_[prompt="what team do you want out check out"],
+                        out league : Entity(com.goal_smart:soccer_league),
+                        out team1: Entity(com.goal_smart:soccer_team),
+                        out team2: Entity(com.goal_smart:soccer_team),
                         out venue: String)
   #_[canonical="team fixtures, team matches, team games"]
   #_[confirmation="fixtures of $team_id"]
   #_[result=["${team1} vs ${team2} at ${venue} for the ${league}"]]
   #[poll_interval=3600000ms]
   #[doc="Get the fixtures of the team"];
 
-  monitorable list query previousFixtures(in req team_id: Entity(com.goal_smart:soccer_team) #_[prompt="what team do you want out check out?"],
-                        out league : String,
-                        out team1: String,
-                        out team2: String,
+  monitorable list query previousFixtures(in req team_id: Entity(com.goal_smart:soccer_team) #_[prompt="what team do you want out check out"],
+                        out league : Entity(com.goal_smart:soccer_league),
+                        out team1: Entity(com.goal_smart:soccer_team),
+                        out team2: Entity(com.goal_smart:soccer_team),
                         out score1: Number,
                         out score2: Number,
+                        out result: String,
```

Comment:
You'll want to use an enum, not a string here. If you use an enum, you can actually filter on it.
Also, with plain string concatenation, translating the skill to other languages typically becomes impossible because it botches the grammar in the other language.

The other comments I had here still apply by the way.